ga4gh/datamodel/reads.py

```diff
@@ -289,6 +300,23 @@ def getNumUnalignedReads(self):
     def getPrograms(self):
         return self._programs
 
+    def getReadAlignments(self, reference, start, end):
+        """
+        Returns an iterator over all reads in a ReadGroupSet
+        """
+        samFile = self.getFileHandle(self._samFilePath)
+        referenceName = reference.getLocalId().encode()
+        start, end = self.sanitizeAlignmentFileFetch(start, end)
+        readAlignments = samFile.fetch(referenceName, start, end)
+        syntheticReadGroup = HtslibReadGroup(self, 'synthetic')
```

Comment:
Confused by this... Is this to get access to the ``readGroup.convertReadAlignment`` method? If so, I think we should consider refactoring so that both classes have access to this function/method.